test/images/image-util.sh

```diff
@@ -136,11 +136,7 @@ build() {
         ${SED} -i "s|QEMUARCH|${QEMUARCHS[$arch]}|g" Dockerfile
         # Register qemu-*-static for all supported processors except the current one
         echo "Registering qemu-*-static binaries in the kernel"
-        local sudo=""
-        if [[ $(id -u) != 0 ]]; then
-          sudo=sudo
-        fi
-        ${sudo} "${KUBE_ROOT}/third_party/multiarch/qemu-user-static/register/register.sh" --reset -p yes
+        docker run --rm --privileged multiarch/qemu-user-static:5.1.0-2 --reset -p yes
         curl -sSL https://github.com/multiarch/qemu-user-static/releases/download/"${QEMUVERSION}"/x86_64_qemu-"${QEMUARCHS[$arch]}"-static.tar.gz | tar -xz -C "${temp_dir}"
```

Comment:
I didn't realize this wasn't checking a hash at least. yikes

Reply:
at least it's versioned, it's using ``QEMUVERSION`` release.